### PR TITLE
Feat/cp 108 add setting page

### PR DESCRIPTION
**Pull request by SeijaHuang**
feat: add setting page
Added account setting page:
- implemented account setting form
- implemented account setting page

Added variant for shared input component:
- added display variant to enable the label and input displaying either horizontally or vertically

Added user help messages and validation messages:
- added user help messages to automatically pop up when clicking responding input field
- added validation messages which are mainly for form validation error messages

Resolve CP-108

Screenshot:
Platform:
light theme
image
image

dark theme
image
image

Turbo:
light theme
image
image
image

dark theme
image
image

### CODE REVIEW

File: apps/client/src/module/common/controlled-input.tsx
Comment (NorrisWu0) [inline apps/client/src/module/common/controlled-input.tsx]: typo here, it's actually `column`.

File: apps/client/src/module/common/controlled-input.tsx
Comment (NorrisWu0) [inline apps/client/src/module/common/controlled-input.tsx]: best to be explicit here... if `display === 'column'` then show this tooltip, then on the other tooltips do `display === 'inline'`

actually, if we doing `column`, maybe we can call this `row`? and maybe instead of `display` we call it `direction`?

File: apps/client/src/module/common/controlled-input.tsx
Comment (NorrisWu0) [inline apps/client/src/module/common/controlled-input.tsx]: feels a little weird that we are showing/hiding tooltips when it's the label that's moving.... but it's working now and I have no complaints!

File: apps/client/src/module/common/controlled-input.tsx
Comment (NorrisWu0) [inline apps/client/src/module/common/controlled-input.tsx:67]: assuming the rest is the same, so not gonna look then

**Review by NorrisWu0 — CHANGES_REQUESTED** [review]

**Comment by NorrisWu0** [comment]
register form also has this tooltip yeah? since you have done this, might as well refactor that one as well?

File: apps/client/src/module/common/controlled-input.tsx
Comment (SeijaHuang) [reply apps/client/src/module/common/controlled-input.tsx]: fixed

**Review by NorrisWu0 — APPROVED** [review]

**Comment by NorrisWu0** [comment]
lgtm feel free to merge when ever

**Review by Chris-Wang — CHANGES_REQUESTED** [review]
The button looks different:
- it adds border
- no anima
- what happens to cancel button?

New:
image
Original:
image

Shadows are different:
New:
image
Original:
image

**Comment by Chris-Wang** [comment]
The buttons get the same issue as the one on Login page, and I create a ticket for that: https://beequant.atlassian.net/browse/CP-112

So, this PR could wait for CP-112 get resolved, and then fix the button here.
Or update the PR without fixing the button, but will require a button fix of setting page in the furture

**Comment by SeijaHuang** [comment]
> The button looks different:
> 
> * it adds border
> * no anima
> * what happens to cancel button?
> 
> New: image Original: image
> 
> Shadows are different: New: image Original: image

The button UI has been updated by Norris. This PR has updated with the latest version of dev to sovle this issue.

**Review by Chris-Wang — CHANGES_REQUESTED** [review]
Issue 1:
- setting page cannot access from topper menu:
image

Please test your PR after merge latest dev before pushing it

**Comment by Chris-Wang** [comment]
BTW did the setting page shadow get updated?

File: apps/client/package.json
Comment (Chris-Wang) [inline apps/client/package.json]: are we using turbo here?

File: apps/client/src/utils/validation-message.ts
Comment (Chris-Wang) [inline apps/client/src/utils/validation-message.ts:1]: if these regex not being used anymore, we can remove it

**Review by Chris-Wang — CHANGES_REQUESTED** [review]

File: apps/client/src/utils/validation-message.ts
Comment (SeijaHuang) [reply apps/client/src/utils/validation-message.ts:1]: use this pattern in the setting form validation

File: apps/client/package.json
Comment (SeijaHuang) [reply apps/client/package.json]: yes, but I will leave this setting change for Norris!

**Review by Chris-Wang — APPROVED** [review]

**Pull request merged**
